packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -20,6 +20,10 @@ import type { MiniflareOptions, ModuleRule, WorkerOptions } from "miniflare";
  * Options for the `getPlatformProxy` utility
  */
 export type GetPlatformProxyOptions = {
+	/**
+	 * The name of the environment to use
+	 */
+	env?: string;
```

Comment:
I'd probably make this `environment` since we use `env` itself in a very defined way already. Just a bit confusing

Reply:
done 🙂👍